[Bug]: libimlib2_old fails to build

### What happened?

After the latest commit [Compile tolua++ output as C++](https://github.com/brndnmtthws/conky/commit/e4bdabc5380ff2d0e164ab0bcda2c3ae7c12010b) the build fails with:
`/var/portage/tmp/portage/app-admin/conky-9999/work/conky-9999_build/lua/libimlib2_old.cc:3680:4: error: ‘imlib_render_image_updates_on_drawable’ was not declared in this scope; did you mean ‘imlib_render_image_on_drawable’?
 3680 |    imlib_render_image_updates_on_drawable(updates,x,y);
      |    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    imlib_render_image_on_drawable
`

Before only a warning was issued:
`/var/portage/tmp/portage/app-admin/conky-9999/work/conky-9999_build/lua/libimlib2_old.c:3680:4: warning: implicit declaration of function ‘imlib_render_image_updates_on_drawable’; did you mean ‘imlib_render_image_on_drawable’? [-Wimplicit-function-declaration]
 3680 |    imlib_render_image_updates_on_drawable(updates,x,y);
      |    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    imlib_render_image_on_drawable
`

### Version

git latest

### Which OS/distro are you seeing the problem on?

Gentoo

### Conky config

_No response_

### Stack trace

_No response_

### Relevant log output

_No response_

--- a/lua/imlib2_old.pkg
+++ b/lua/imlib2_old.pkg
@@ -1,3 +1,4 @@
+$#define BUILD_X11 1
 $#include<Imlib2.h>
 $#include<X11/Xlib.h>
 $#define _userdata void *

